x-pack/plugins/monitoring/public/services/clusters.js

```diff
@@ -55,10 +55,10 @@ export function monitoringClustersProvider($injector) {
     }
 
     function ensureAlertsEnabled() {
-      return $http.post('../api/monitoring/v1/alerts/enable', {}).catch((err) => {
-        const Private = $injector.get('Private');
-        const ajaxErrorHandlers = Private(ajaxErrorHandlersProvider);
-        return ajaxErrorHandlers(err);
+      return $http.post('../api/monitoring/v1/alerts/enable', {}).catch(() => {
+        /**
+         * Ignoring for now, but should really indicate what is the cause
```

Comment:
I we should at least show a toast saying something like:

Title: 'Unable to enable alerts'
Body: 'TLS is required for alerting and must be enabled in kibana.yml`

Or just borrow what the alerting team has in https://github.com/elastic/kibana/blob/master/x-pack/plugins/triggers_actions_ui/public/application/components/health_check.tsx